cloud-azure/build.gradle

```diff
@@ -29,6 +29,7 @@ dependencies {
   compile (group: 'com.microsoft.azure',         name: 'azure-mgmt-sql',             version: azureSdkVersion) { exclude group: 'org.slf4j' }
   compile group: 'com.microsoft.azure',         name: 'azure-storage',              version: '6.1.0'
   compile group: 'com.microsoft.azure',         name: 'adal4j',                     version: '1.3.0'
+  compile group: 'org.apache.commons',                 name: 'commons-collections4', version: '4.3'
```

Comment:
do we really need this dependency?

Reply:
yes, we're using this dependency but after upgrading the azure sdk version it is gone. probably the 1.4 has contained it but the newer doesn't.